timemachine/fe/absolute_hydration.py

```diff
@@ -219,6 +220,9 @@ def estimate_absolute_free_energy(
     steps_per_frame: int
         The number of steps to take before collecting a frame
 
+    image_traj: bool
+        Images the trajectories returned with the SimulationResult
```

Comment:
nit: I think it'd be helpful to briefly describe what imaging means here

Reply:
Fixed in [59f52c1](https://github.com/proteneer/timemachine/pull/905/commits/59f52c120f00e9e81a6bcca2b3eecccc36ac75d9)